TunnelVisionLabs.ReferenceAssemblyAnnotator/Program.cs

```diff
@@ -93,6 +106,16 @@ private static void AnnotateType(TaskLoggingHelper? log, TypeDefinition typeDefi
             }
 
             Annotate(typeDefinition, annotatedTypeDefinition, attributesOfInterest);
+            for (var i = 0; i < typeDefinition.Interfaces.Count; i++)
+            {
+                for (int j = 0; j < annotatedTypeDefinition.Interfaces.Count; j++)
+                {
+                    if (EquivalenceComparers.TypeReference.Equals(typeDefinition.Interfaces[i].InterfaceType, annotatedTypeDefinition.Interfaces[j].InterfaceType))
+                    {
+                        Annotate(typeDefinition.Interfaces[i], annotatedTypeDefinition.Interfaces[j], attributesOfInterest);
```

Comment:
On phone so maybe I'm missing it, but are you annotating typeDefinition.BaseType?

Reply:
Wasn't thinking. BaseType isn't a custom attribute target, so it's gotta be handled by the annotation on the type itself.